pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -100,6 +100,10 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, c *ConverterContext)
 				}
 			}
 		}
+
+		if iface.State == v1.InterfaceStateLinkDown {
+			domainIface.LinkState = &api.LinkState{State: "down"}
+		}
```

Comment:
@nirdothan Unless these are implementation details, I would recommend to unit test it. As per my understanding from the commit message, if the network interface is SRIOV we cannot expect `CreateDomainInterfaces` to set the link to down.

Reply:
@nirdothan Thank you for the clarification.